tools/topology/topology2/cavs/CMakeLists.txt

```diff
@@ -25,6 +25,9 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt.bin"
 
 # CAVS SDW topology with passthrough pipelines
 "cavs-sdw\;cavs-sdw"
+# CAVS SDW + DMIC topology with passthrough pipelines
+"cavs-sdw\;cavs-mtl-sdw-dmic\;NUM_DMICS=4,\
+PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-cavs-mtl-nocodec.bin"
```

Comment:
what does this reference to nocodec have to do with SoundWire?
@ranj063 @juimonen ?

Reply:
> what does this reference to nocodec have to do with SoundWire? @ranj063 @juimonen ?

@plbossart nhlt-cavs-mtl-nocodec.bin also include the DMIC blob. @ranj063 @juimonen Please correct me if I was wrong.

Reply:
It seems we have a little ambiguity over which DAIs the "nocodec" label applies to here. @plbossart I assume that SDW does not support nocodec mode ?
@bardliao The CMakefile should build the NHLT_BIN file using regular Cmake rules so that any changes to the NHLT would automatically cause is users to be rebuilt.

Reply:
I don't understand what this 'nocodec' might mean for SoundWire.

Reply:
@lgirdwood @plbossart Thanks to @ranj063 I finally figure out what is the nhlt bin used for. I renamed it to "nhlt-mtl-sdw-dmic.bi" now.